pkg/e2e/metrics.go

```diff
@@ -230,11 +230,7 @@ func (m *Metrics) jsonToPrometheusOutput(gatherer *prometheus.GaugeVec, phase st
 
 			// We're only concerned with tracking float values in Prometheus as they're the only thing we can measure
 			if floatValue, err := strconv.ParseFloat(stringValue, 64); err == nil {
-				if phase != "" {
```

Comment:
Since my reading yielded "empty label values are A-OKAY" we can simplify this to just always pass through phase.

Reply:
So I'm not sure about this -- basically, `cicd_metadata` isn't supposed to report the phase as it exists outside of any install/upgrade phase. It looks like phase is inserted above, which (IMO) is not desirable.

Reply:
I get what you're layin' down. 

Is the same true for the Addon Prometheus object?

Reply:
So actually no, at least IMO. The addon metadata could potentially be captured in the install or upgrade phases. We don't currently have any upgrade addon tests, but they could potentially exist.